submissions/Sveti-Stefan/hooli-style-popup/styles.css

```diff
@@ -0,0 +1,196 @@
+body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+.container {
+  display: block;
+  width: 100%;
+  height: 100px;
+  background-color: #0c86df;
+}
+.container_header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100px;
+  width: 100%;
+}
+.logo_header {
+  display: flex;
+  align-content: center;
+  height: 50px;
+}
+.menu_header_nav {
+  display: flex;
+  height: 100px;
+  align-items: center;
+}
+.menu_header_list {
+  display: flex;
+  list-style-type: none;
+  justify-content: flex-end;
+  margin: 0px;
+}
+.menu_header_list_item {
+  padding: 40px;
+  color: white;
+  text-decoration: none;
+}
+
+.menu_header_list_item a {
+  color: white;
+  text-decoration: none;
+  font-size: 24px;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.popup_checkbox {
+  appearance: none;
+}
+
+.popup_checkbox:checked ~ .popup {
+  display: block;
+}
+
+.popup {
+  display: none;
+  position: absolute;
+  background-color: #fff;
+  border-radius: 10px;
+  border: #0c86df solid 2px;

```

Comment:
The convention is typically `[width] [style] [color]` but this doesn't matter.

Reply:
Fixed